notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p><code>&gt; stale-test-threshold&nbsp;probably indicates the threshold for the number of tests that can be stale for a particular job.</code></p><p> </p><p>I think it probably means the number of times a test is not run in a job before it gets labeled as "stale" and hidden in the UI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>